src/main/java/mate/academy/intro/model/Book.java

```diff
@@ -0,0 +1,22 @@
+package mate.academy.intro.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Entity
+@Data
```

Comment:
Don't use `@Data` for entities - it can cause some problems
Also, add `@Table` 

```suggestion
@Getter
@Setter
```